src/webview-ext-common.ts

```diff
@@ -345,9 +339,8 @@ export class WebViewExtBase extends ContainerView {
 
     /**
      * Gets the native [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview/) that represents the user interface for this component. Valid only when running on iOS 11+.
-     * Gets the native [UIWebView]https://developer.apple.com/documentation/uikit/uiwebview that represents the user interface for this component. Valid only when running on iOS <11
      */
-    public ios: any /* WKWebView | UIWebView */;
+    public ios: any /* WKWebView */;
```

Comment:
could just give it the WKWebView type now.

Reply:
Native type is usually not exported, due to the size of the typings needed.

Reply:
👍